File: chapter4/file_pointers/phonebook.c
// Save names and numbers to a CSV file

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Open a .csv file
    FILE *fonebook = fopen("phonebook.csv", "a");

    // Check to see if file opened successfully
    if (!fonebook)
    {
        printf("Error loading file\n");
        return 1;
    }

    // Get name and number
    char *name = get_string("Name: ");
    char *number = get_string("Number: ");

    // Write to the file
    fprintf(fonebook, "%s,%s\n", name, number);

    // Close the file
    fclose(fonebook);
}

File: chapter5/csv/phonebook.py
import csv
from cs50 import get_string

# Open phonebook.csv in "a"ppend mode
file = open("phonebook.csv", "a")

# Get number from the user
name = get_string("Name: ")
number = get_string("Number: ")

# Write the name and number for the file
writer = csv.writer(file)
writer.writerow([name, number])

# Close the open file
file.close()
